docker/start.sh

```
@@ -0,0 +1,94 @@
+
+trap "cleanup; exit 0" EXIT
+trap "cleanup; exit 130" INT
+trap "cleanup; exit 143" TERM
+
+print_header "3. Configuring Azure Pipelines agent..."
+
+./config.sh --unattended \
+  --agent "${AZP_AGENT_NAME:-$(hostname)}" \
+  --url "${AZP_URL}" \
+  --auth "PAT" \
+  --token $(cat "${AZP_TOKEN_FILE}") \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)